app/Resources/views/search/layout.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
+
```

Comment:
Ok this is why naming is important, so this template is `search/index.html.twig` and the other one should be included here (instead of extending) and prefixed with an underscore `search/_committees.html.twig`